### PR TITLE
Individual Membership registration and election

### PR DESCRIPTION
**Pull request by mikeal**
Would love some copy editing support from @nodejs/evangelism and a double-check on the terms from @nodejs/membership .

### CODE REVIEW

**Comment by julianduque** [comment]
Copy looks good, do you think it is a good idea to turn company names to links?

**Comment by fhemberger** [comment]
LGTM. I'm don't think we need company links here.

**Comment by dshaw** [comment]
Fantastic. Glad to see individual membership finally roll out.

File: locale/en/blog/community/individual-membership.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Who can run for the board of directors?
+
+Any registered member.
+
+Keep in mind that every meeting of the Board must reach quorum in order to pass resolutions, so only people who can make themselves available on a recurring and consistent basis should consider running.
+
+## What does the Board of Directors do?
+
+The Board meets every month to approve resolutions and discuss Node.js Foundation administrative matters. This includes legal considerations, budgetting and approving Foundation-lead conferences and other initiatives. Technical governance is overseen by the TSC, not the Board of Directors.
```
Comment (thefourtheye) [inline locale/en/blog/community/individual-membership.md]: `Foundation-led`? Also, TSC is the new CTC, right? cc @nodejs/tsc

File: locale/en/blog/community/individual-membership.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## When is the election?
+
+* Nominations are being solicited until January 15th.
+* A ballot will be distributed on January 20th.
+* The election will be completed by January 30th.
+
+## How do I run in the 2016 election?
+
+After you've registered as an a member follow the instructions [here](https://github.com/nodejs/membership/issues/12).
```
Comment (thefourtheye) [inline locale/en/blog/community/individual-membership.md]: `as a member,`

File: locale/en/foundation/board.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: Node.js Foundation Board of Directors
+layout: foundation.hbs
+---
+
+The Board of Directors is composed of representatives from corporate members, a representative of the
+Technical Steering Committee, and soon representatives elected by the individual membership class.
+
+* Chairperson: **Danese Cooper**, distinguished member of technical staff - open source at Paypal.
```
Comment (thefourtheye) [inline locale/en/foundation/board.md]: Isn't it `PayPal`?

File: locale/en/blog/community/individual-membership.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+status: publish
+category: Community
+slug: individual-membership-nodejs-foundation
+layout: blog-post.hbs
+author: mikeal
+---
+
+The Node.js Foundation is a member-supported organization. To date we've added over 20 corporate members who provide the financial support necessary for the Foundation to thrive.
+
+With the support of the Linux Foundation we are now able to launch an Individual Membership program. These members will be electing two representatives to the Board of Directors this January who will be
```
Comment (thefourtheye) [inline locale/en/blog/community/individual-membership.md:13]: some commas around `this January`?

**Pull request merged**